.gitignore

```diff
@@ -1,3 +1,4 @@
 .replit
 workspace.code-workspace
 temp.py
+.DS_Store
```

Comment:
I don't understand this change

Reply:
The .DS_Store file, is a temporal file created by macOS, so i don't think it should be included in the git repository